src/DocsGenerator/Helpers/CategoryHelper.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Codacy.Engine.Seed.Patterns;
+
+namespace Codacy.TSQLLint.DocsGenerator.Helpers
+{
+    /// <summary>
+    /// Helper to convert to codacy category from a TSQLLint rule
+    /// </summary>
+    public static class CategoryHelper
+    {
+        /// <summary>
+        /// This map a rule to a codacy category.
+        /// </summary>
+        /// <param name="rule">rule name</param>
+        /// <returns>Codacy category</returns>
+        /// <exception cref="NotImplementedException">When found a non explicit mapped rule</exception>
+        public static Category ToCategory(string rule)
+        {
+            switch (rule)
+            {
+                case "conditional-begin-end": return Category.ErrorProne;

```

Comment:
can we divide categories in folder other than having switch cases in code? cc. @lolgab

Reply:
I don't get the point. 

Btw, I can also make an improvement here, instead of making a lot of repeated code with returns, i can just return at the end of the same category `case`, as suggested by @lolgab .

Reply:
I've done the improvement I mentioned.